src/main/resources/mappings/workflow-steps.json

```diff
@@ -39,18 +39,34 @@
             "connector_id"
         ]
     },
-    "register_model": {
+    "register_local_model": {
         "inputs":[
-            "function_name",
             "name",
-            "description",
```

Comment:
We can include the description field here. Or are we just including the required fields here for validation?

Reply:
these are only the required fields